src/libuv/Native.Settings

```diff
@@ -50,6 +50,8 @@
     <Link>
       <EnableCOMDATFolding>true</EnableCOMDATFolding>
       <OptimizeReferences>true</OptimizeReferences>
+      <ControlFlowGuard>Guard</ControlFlowGuard>
+      <DebugInformationFormat>None</DebugInformationFormat>
```

Comment:
I got this from a C++ project in VS, it prevents `/ZI` switch from being used, I could probably change this to `ProgramDatabase` which is `/Zi`